opacus/accountants/utils.py

```diff
@@ -1,24 +1,18 @@
 from opacus.accountants import create_accountant
 
 
-# min range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MIN_BOUND = 0.01
-# starting point for a max range bound when searching for sigma given epsilon
-DEFAULT_SIGMA_MAX_BOUND = 10
-# condition to halt binary search for sigma given epsilon
-SIGMA_PRECISION = 0.01
-# max possible value for returned sigma.
-# Noise higher than MAX_SIGMA considered unreasonable
-MAX_SIGMA = 2000
-
-
 def get_noise_multiplier(
     *,
     target_epsilon: float,
     target_delta: float,
     sample_rate: float,
-    epochs: int,
+    epochs: int = None,
+    steps: int = None,
     accountant: str = "rdp",
+    sigma_precision=0.01,
```

Comment:
You need to add check that `sigma_tolerance` is strictly positive. Alternative: pass is to scipy.bisect and let it do the check for you.